sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/RntbdTransportClient.java

```diff
@@ -355,6 +370,26 @@ private void throwIfClosed() {
         }
     }
 
+    public EnumSet<MetricCategory> getMetricCategories() {
+        return this.metricConfig != null ?
+            ImplementationBridgeHelpers
+                .CosmosClientTelemetryConfigHelper
+                .getCosmosClientTelemetryConfigAccessor()
+                .getMetricCategories(this.metricConfig) : MetricCategory.DEFAULT_CATEGORIES;
```

Comment:
why we are returning default categories if the client telemetry is not configured at all?